### DISCUSSION TITLE
Unwanted and unsupported supports?

### DISCUSSION THREAD

**Discussion by macdarren** [supermerill/SuperSlicer#1525] [Q&A]
I have a model that generally requires some support.
It mostly works as expected.
I have tried a variety of settings and gotten supports I can remove and mostly provide the support required.

However I needed to block support in a few areas as it is just not needed and is hard to remove.
I have tried the built in support blockers.
I have also created a mask to block supports with similar results.
The problem is I get these little unsupported towers printed in mid-air.
They tend to cause a build up of filament that snag on the hot end.

I am using the latest nightly build for MacOS, and have included the project file.

Hoping for advice on how to block the floating support material created
(not sure why it is created it doesn't actually support anything even if it went down to the build platform)
I think it is supposed to be for the row of 'windows' at the bottom of the model...
I would prefer not to try to create support enforcers as I have other models with similar issues, that would be a bit more complex to enforce...prefer to just block or figure why these supports are being created and turn them off.

Thanks in advance and thanks to the creator and people working to advance this powerful slicer.
[board mount 7-6-21l.3mf.zip](https://github.com/supermerill/SuperSlicer/files/7094679/board.mount.7-6-21l.3mf.zip)

**Comment by supermerill** [comment]
Where are the problematic areas?

Your blockers seems to work. 

![image](https://user-images.githubusercontent.com/6536403/132107198-ee358f53-fe19-4c19-ad19-8079e146f623.png)

**Comment by macdarren** [comment]
the view you show it is hard to see...see attached:
![Untitled](https://user-images.githubusercontent.com/25966982/132118905-c8182c14-33c6-44e6-9da6-404eec489e97.jpg)

**Comment by macdarren** [comment]
another angle from a higher layer
![Screen Shot 2021-09-05 at 12 27 28 AM](https://user-images.githubusercontent.com/25966982/132119091-87860a79-a7c7-40b4-91e1-73d0367dd76e.jpg)

**Reply by supermerill** [reply]
To create the pillar, it takes the bottom surfaces that need support and expands them to have wide enough support pillars.
But it doesn't check for their validity, and this thing is a renmant of the pillar that is far enough away of the part to be "valid".
You may be able to remove it by increasing "XY separation between an object and its support" a bit.

**Comment by macdarren** [comment]
Ok so increasing XY separation to 2.5mm from 2mm gets rid of the unwanted towers.  Should there be a request to have the towers checked against the support blockers.  Never having done a formal request I am not really sure how it where to file it.   Thanks for the assist and great software.
